sql/statesql/schema/migrations/0024_malfeasance.sql

```diff
@@ -0,0 +1,66 @@
+CREATE TABLE marriages
+(
+    pubkey          CHAR(32) PRIMARY KEY,   -- pubkey of the identity in the marriage set
+    id              INT NOT NULL,           -- id of the marriage set
+    marriage_atx    CHAR(32) NOT NULL,      -- marriage atx of the identity (can differ within set because of double marriage)
+    marriage_idx    INT NOT NULL,           -- index of the identity in the marriage set
+    marriage_target CHAR(32) NOT NULL,      -- target of the marriage certificate (pubkey of identity that signed marriage atx)
+    marriage_sig    BLOB NOT NULL,          -- proof of the identity in the marriage set
```

Comment:
I will need those two fields for the new malfeasance sync. The idea is that a peer can request a proof for Alice and I can serve them a proof for Bob as long as I also include the necessary certificates to show that Alice and Bob were married. For this I need those two fields.

They are "duplicated" from the ATXs because I want to make malfeasance proofs self contained - this would allow nodes to prune ATXs from identities they consider malicious (as long as they are not referenced somewhere else).